ingest-app/docs/api.md

```diff
@@ -1169,6 +1169,10 @@ For NRT Notification subscriptions to be used there are three new fields that ar
         </ul>
 </ul>
 
+**IMPORTANT: There is limit to creating and updating subscriptions related to the specific scenario highlighted below:
+Currently, if User1 creates a subscription to Coll1 and then User2 creates a different subscription to the same Coll1 for the same queue, it will overwrite the first subscription for that specific queue.
```

Comment:
User1 rubs me the wrong way, only because people complain about our API docs a lot. While I am probably responsible for a good number of documentation complaints (having created a lot of them) I feel like we could do something better here. What about `"<User One>"` and `"<Collection One>"`?

Reply:
I also do not like the use of ** for markdown when it has no markdown meaning. This could confuse bad parsers.